packages/blade/src/components/Button/_decisions/decisions.md

```diff
@@ -0,0 +1,81 @@
+# Button Decisions <!-- omit in toc -->
+- [Design](#design)
+- [Internal BaseButton Component](#internal-basebutton-component)
+- [Button Component](#button-component)
+- [Component Breakdown](#component-breakdown)
+- [Open Questions](#open-questions)
+
+## Design
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=5200%3A0) to all variants of the Button component
+
+## Internal BaseButton Component
+-  Internal component that exposes certain extra props to enable creation of Component Level Action Buttons like `positive`, `negative`, `neutral`
+-  **BaseButton API**:
+   -  **`variant`**: `primary` | `secondary` | `tertiary` | `link`
+      -  **default**: `primary`
+      -  Link will look like an inline Link component but will have the role of a `button`
```

Comment:
> component but will have the role of a `button`

Do we mean it will be `<a>` but with role="button"? or will it be rendered as `<button />`?

Reply:
It will always render as button.

Reply:
aha nice!

Reply:
```suggestion
      -  Link will look like an inline Link component but will be rendered as `button` element
```

can we rephrase this? got a bit confused because of `role of a button` line